### ISSUE TITLE
Confiscation: TypeError: Cannot read property 'owner' of undefined

### ISSUE THREAD

**Issue by cryogen** [cryogen/throneteki#692]
https://sentry.io/throneteki/throneteki/issues/241341857/

```
TypeError: Cannot read property 'owner' of undefined
  File "/home/stu/throneteki/server/game/cards/plots/01/confiscation.js", line 12, in CardForcedInterrupt.handler
    attachment.owner.discardCard(attachment);
  File "/home/stu/throneteki/server/game/forcedtriggeredability.js", line 34, in CardForcedInterrupt.executeHandler
    if(this.handler(context) !== false && this.limit) {
  File "/home/stu/throneteki/server/game/gamesteps/abilityresolver.js", line 102, in AbilityResolver.executeHandler
    this.ability.executeHandler(this.context);
  File "/home/stu/throneteki/server/game/gamesteps/abilityresolver.js", line 20, in SimpleStep.AbilityResolver.pipeline.initialise.SimpleStep [as continueFunc]
    new SimpleStep(game, () => this.executeHandler())
  File "/home/stu/throneteki/server/game/gamesteps/simplestep.js", line 10, in SimpleStep.continue
    return this.continueFunc();
...
(17 additional frame(s) were not displayed)

TypeError: Cannot read property 'owner' of undefined
```

**Comment by cryogen** [comment]
@ystros context.target was undefined in the handler for confiscation - do you know of cases where/why this might happen?

**Comment by ystros** [comment]
@cryogen I suspect it's similar to cases we've seen where a select card prompt resulted in a player selecting something that's not a card, but I've never been able to reproduce those cases.

It definitely isn't the case when the player clicks 'Done' without selecting an attachment. That cancels the resolution of Confiscation as expected / intended.

**Comment by cryogen** [comment]
https://sentry.io/throneteki/throneteki/issues/260952297/

**Comment by cryogen** [comment]
Fixed in 805909f8fcc0057c79bae956e8b12b722bb87f07
